spec/support/ansible_shared/automation_manager/credential.rb

```diff
@@ -1,8 +1,8 @@
 require 'ansible_tower_client'
 
-shared_examples_for "ansible credential" do
+shared_examples_for "ansible credential" do |ansible_provider|
   let(:finished_task) { FactoryGirl.create(:miq_task, :state => "Finished") }
-  let(:manager)       { FactoryGirl.create(:provider_ansible_tower, :with_authentication).managers.first }
+  let(:manager)       { FactoryGirl.create(ansible_provider, :with_authentication).managers.first }
```

Comment:
Yeah, but as I said, that is passing the parameter `implicitly` and I prefer more `explicit` passing which brings more clarity to the code.